WordPress/src/main/res/layout/stats_site_selector.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingTop="16dp">
```

Comment:
Should we use common values (eg. `margin_extra_large`) from `dimens.xml` here and in `stats_site_selector_item` ?